charts/kube-master/charts/etcd/templates/deployment.yaml

```diff
@@ -42,33 +76,60 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: status.podIP
-          command: [ etcd ]
-          args:
-            - --name=kubernikus
-            - --data-dir=/var/lib/etcd
-            - --advertise-client-urls=http://$(ETCD_IP):2379
-            - --initial-advertise-peer-urls=http://$(ETCD_IP):2380
-            - --initial-cluster=kubernikus=http://$(ETCD_IP):2380
-            - --listen-client-urls=http://0.0.0.0:2379
-            - --listen-peer-urls=http://$(ETCD_IP):2380
-            {{- if false }}
-            - --trusted-ca-file=/etc/etcd/certs/etcd-clients/ca.pem
-            - --cert-file=/etc/etcd/certs/etcd-clients/etcd-server.pem
-            - --key-file=/etc/etcd/certs/etcd-clients/etcd-server-key.pem
-            - --peer-ca-file=/etc/etcd/certs/etcd-peers/ca.pem
-            - --peer-cert-file=/etc/etcd/certs/etcd-peers/etcd-0.pem
-            - --peer-key-file=/etc/etcd/certs/etcd-peers/etcd-0-key.pem
-            {{- end }}
+          command:
+            - /bootstrap/bootstrap.sh
           volumeMounts:
-            {{- if false }}
-            - mountPath: /etc/etcd/certs/etcd-clients
-              name: etcd-clients
-              readOnly: true
-            - mountPath: /etc/etcd/certs/etcd-peers
-              name: etcd-peers
-              readOnly: true
-            {{- end }}
             - mountPath: /var/lib/etcd
               name: data
+            - mountPath: /bootstrap
+              name: {{ include "fullname" . }}-bootstrap
           resources:
+        - name: backup
+          command:
+            - etcdbrctl
+            - server
+            - --schedule=*/5 * * * *
```

Comment:
Any idea what the storage requirements are?

Reply:
Hard to tell, this depends on general etcd data amount and utilization/changes in data.